extensions/webgpu/swapchain.rs

```diff
@@ -0,0 +1,141 @@
+use crate::WebGpuResult;
+use deno_core::error::bad_resource_id;
+use deno_core::error::AnyError;
+use deno_core::OpState;
+use deno_core::Resource;
+use deno_core::ResourceId;
+use deno_core::ZeroCopyBuf;
+use serde::Deserialize;
+use std::borrow::Cow;
+
+struct WebGpuRawWindowHandle(DynHasRawWindowHandle);
```

Comment:
This struct shouldn't be public? I mean, how can I integrate with Winit Window with this struct? When I tried to use the Winit window this error occurred:

> `error: Uncaught (in promise) BadResource: Bad resource ID`